test/test_habitat_env.py

```diff
@@ -132,7 +134,9 @@ def test_threaded_vectorized_env():
     envs = habitat.ThreadedVectorEnv(env_fn_args=env_fn_args)
     envs.reset()
     non_stop_actions = [
-        v for v in range(len(SimulatorActions)) if v != SimulatorActions.STOP
+        act
```

Comment:
Looks like we could reduce copypasta using util function: `non_stop_actions(env)`